dotcom-rendering/src/palette.ts

```diff
@@ -4871,6 +4896,8 @@ const interactiveAtomBackgroundDark: PaletteFunction = () =>
 /**
  * A template literal type used to make sure the keys of the palette use the
  * correct CSS custom property syntax.
+ *
+ * IMPORTANT - do use a CSS variable name ending in "label" as Storybook theming will fail
```

Comment:
should this be `do not use` ?

```suggestion
 * IMPORTANT - do not use a CSS variable name ending in "label" as Storybook theming will fail
```